Gemfile.lock

```diff
@@ -41,7 +41,7 @@ GEM
     unicode-display_width (2.5.0)
 
 PLATFORMS
-  arm64-darwin-23
+  ruby
```

Comment:
I think this is what did it, based on the earlier error message trying to use `bundler: none`:

```
The dependency memo_pad (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for arm64-darwin-23 but the dependency is only for ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, ruby, . To add those platforms to the bundle, run `bundle lock --add-platform ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby ruby `.
Unable to find a spec satisfying memo_pad (>= 0) in the set. Perhaps the
lockfile is corrupted?
```

I'm not sure what set that, maybe `bundler init`? But we want to install on any Ruby deployment, not just Mac on ARM.